src/Sarif.Multitool/Rules/SARIF2004.OptimizeFileSize.cs

```diff
@@ -23,12 +24,19 @@ public class OptimizeFileSize : SarifValidationSkimmerBase
         public override MultiformatMessageString FullDescription => new MultiformatMessageString { Text = RuleResources.SARIF2004_OptimizeFileSize_FullDescription_Text };
 
         protected override IEnumerable<string> MessageResourceNames => new string[] {
-                    nameof(RuleResources.SARIF2004_OptimizeFileSize_Warning_EliminateLocationOnlyArtifacts_Text)
+                    nameof(RuleResources.SARIF2004_OptimizeFileSize_Warning_EliminateLocationOnlyArtifacts_Text),
+                                        nameof(RuleResources.SARIF2004_OptimizeFileSize_Warning_EliminateIdOnlyRules_Text)
```

Comment:
>                     [](start = 20, length = 20)

Alignment. Tabs/spaces issue?

Reply:
eww.. thanks. yes 

---
In reply to: [447981759](https://github.com/microsoft/sarif-sdk/pull/1950#discussion_r447981759) [](ancestors = 447981759)